### ISSUE TITLE
Can not fetch the package with a insecure protocol from my custom import path

### ISSUE THREAD

**Issue by hyper-carrot** [constabulary/gb#358]
Please see contents below:
```bash
$ gb vendor fetch go.wepiao.com/base_lib/log
skipping insecure protocol: http://go.wepiao.com/base_lib/log?go-get=1
FATAL command "fetch" failed: unable to determine remote metadata protocol
FATAL command "vendor" failed: exit status 1
```

As you can see, It is failed to get the package from our git server, but `go get -insecure go.wepiao.com/base_lib/log` is OK.

I think that gb should support `-insecure ` flag.

**Comment by hyper-carrot** [comment]
OK, I found `-precaire `, but it is still fail:

```bash
$ gb vendor fetch -precaire go.wepiao.com/base_lib/log
FATAL command "fetch" failed: unable to determine remote metadata protocol
FATAL command "vendor" failed: exit status 1
```

**Comment by davecheney** [comment]
@hyper-carrot I tried to check the metadata for that url, but it resolves to an rfc1918 address, nobody on the internet will be able to reach that server
```
lucky(~) % curl -v http://go.wepiao.com/base_lib/log
* Hostname was NOT found in DNS cache
*   Trying 192.168.101.48...
```
If this is deliberate, can you post the results of the command above. I only need to see the contents of `<head />`, specifically the `<meta>` tags.

**Comment by hyper-carrot** [comment]
I  found the problem, but gb didn't print it out. I improved it, see #359.

**Comment by hyper-carrot** [comment]
@davecheney  `go.wepiao.com` is a internal url, but the problem really is associated with it.

**Comment by davecheney** [comment]
Closed in 716b8dc
